<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="LTP11" failures="2" tests="14" name="AASProductTestCase.AASProductTest.qa.testcases.ExtraPTPageTest" time="84.835" errors="0" timestamp="17 Aug 2020 03:57:58 GMT" skipped="0">
  <testcase classname="AASProductTestCase.AASProductTest.qa.testcases.ExtraPTPageTest" name="ExtraPTTest" time="5.996"/>
  <testcase classname="AASProductTestCase.AASProductTest.qa.testcases.ExtraPTPageTest" name="ExtraPTTest" time="6.139"/>
  <testcase classname="AASProductTestCase.AASProductTest.qa.testcases.ExtraPTPageTest" name="ExtraPTTest" time="6.003"/>
  <testcase classname="AASProductTestCase.AASProductTest.qa.testcases.ExtraPTPageTest" name="ExtraPTTest" time="5.992"/>
  <testcase classname="AASProductTestCase.AASProductTest.qa.testcases.ExtraPTPageTest" name="ExtraPTTest" time="6.044"/>
  <testcase classname="AASProductTestCase.AASProductTest.qa.testcases.ExtraPTPageTest" name="ExtraPTTest" time="6.003"/>
  <testcase classname="AASProductTestCase.AASProductTest.qa.testcases.ExtraPTPageTest" name="ExtraPTTest" time="6.166"/>
  <testcase classname="AASProductTestCase.AASProductTest.qa.testcases.ExtraPTPageTest" name="ExtraPTTest" time="5.989"/>
  <testcase classname="AASProductTestCase.AASProductTest.qa.testcases.ExtraPTPageTest" name="ExtraPTTest" time="6.077"/>
  <testcase classname="AASProductTestCase.AASProductTest.qa.testcases.ExtraPTPageTest" name="ExtraPTTest" time="6.100"/>
  <testcase classname="AASProductTestCase.AASProductTest.qa.testcases.ExtraPTPageTest" name="ExtraPTTest" time="6.177"/>
  <testcase classname="AASProductTestCase.AASProductTest.qa.testcases.ExtraPTPageTest" name="ExtraPTTest" time="6.062">
    <failure message="expected [999] but found [10.50]" type="java.lang.AssertionError">
      <![CDATA[java.lang.AssertionError: expected [999] but found [10.50]
	at org.testng.Assert.fail(Assert.java:93)
	at org.testng.Assert.failNotEquals(Assert.java:512)
	at org.testng.Assert.assertEqualsImpl(Assert.java:134)
	at org.testng.Assert.assertEquals(Assert.java:115)
	at org.testng.Assert.assertEquals(Assert.java:189)
	at org.testng.Assert.assertEquals(Assert.java:199)
	at AASProductTestCase.AASProductTest.qa.testcases.ExtraPTPageTest.ExtraPTTest(ExtraPTPageTest.java:93)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.base/java.lang.reflect.Method.invoke(Method.java:564)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:108)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:661)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:869)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1193)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:126)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
	at org.testng.TestRunner.privateRun(TestRunner.java:744)
	at org.testng.TestRunner.run(TestRunner.java:602)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:380)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:375)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:340)
	at org.testng.SuiteRunner.run(SuiteRunner.java:289)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1301)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1297)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1226)
	at org.testng.TestNG.runSuites(TestNG.java:1144)
	at org.testng.TestNG.run(TestNG.java:1115)
	at org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:283)
	at org.apache.maven.surefire.testng.TestNGXmlTestSuite.execute(TestNGXmlTestSuite.java:75)
	at org.apache.maven.surefire.testng.TestNGProvider.invoke(TestNGProvider.java:120)
	at org.apache.maven.surefire.booter.ForkedBooter.invokeProviderInSameClassLoader(ForkedBooter.java:384)
	at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:345)
	at org.apache.maven.surefire.booter.ForkedBooter.execute(ForkedBooter.java:126)
	at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:418)
]]>
    </failure>
  </testcase> <!-- ExtraPTTest -->
  <testcase classname="AASProductTestCase.AASProductTest.qa.testcases.ExtraPTPageTest" name="ExtraPTTest" time="6.048"/>
  <testcase classname="AASProductTestCase.AASProductTest.qa.testcases.ExtraPTPageTest" name="ExtraPTTest" time="6.039">
    <failure message="expected [] but found [10.50]" type="java.lang.AssertionError">
      <![CDATA[java.lang.AssertionError: expected [] but found [10.50]
	at org.testng.Assert.fail(Assert.java:93)
	at org.testng.Assert.failNotEquals(Assert.java:512)
	at org.testng.Assert.assertEqualsImpl(Assert.java:134)
	at org.testng.Assert.assertEquals(Assert.java:115)
	at org.testng.Assert.assertEquals(Assert.java:189)
	at org.testng.Assert.assertEquals(Assert.java:199)
	at AASProductTestCase.AASProductTest.qa.testcases.ExtraPTPageTest.ExtraPTTest(ExtraPTPageTest.java:93)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.base/java.lang.reflect.Method.invoke(Method.java:564)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:108)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:661)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:869)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1193)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:126)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
	at org.testng.TestRunner.privateRun(TestRunner.java:744)
	at org.testng.TestRunner.run(TestRunner.java:602)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:380)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:375)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:340)
	at org.testng.SuiteRunner.run(SuiteRunner.java:289)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1301)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1297)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1226)
	at org.testng.TestNG.runSuites(TestNG.java:1144)
	at org.testng.TestNG.run(TestNG.java:1115)
	at org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:283)
	at org.apache.maven.surefire.testng.TestNGXmlTestSuite.execute(TestNGXmlTestSuite.java:75)
	at org.apache.maven.surefire.testng.TestNGProvider.invoke(TestNGProvider.java:120)
	at org.apache.maven.surefire.booter.ForkedBooter.invokeProviderInSameClassLoader(ForkedBooter.java:384)
	at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:345)
	at org.apache.maven.surefire.booter.ForkedBooter.execute(ForkedBooter.java:126)
	at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:418)
]]>
    </failure>
  </testcase> <!-- ExtraPTTest -->
</testsuite> <!-- AASProductTestCase.AASProductTest.qa.testcases.ExtraPTPageTest -->
